auth-server/src/services/auth.ts

```diff
@@ -124,12 +125,12 @@ export default class AuthService {
 			.first()
 
 		if (!user) {
-			throw new Error('User not found')
+			throw new ForbiddenError('User not found')
```

Comment:
Shouldn't this rather be a `AuthenticationError` because it comes from the token?